shell/detail/cis.cattle.io.clusterscan.vue

```diff
@@ -30,7 +32,16 @@ export default {
   },
 
   async fetch() {
-    this.clusterReports = await this.value.getReports();
+    const inStore = this.$store.getters['currentProduct'].inStore;
+    const schema = this.$store.getters[`${ inStore }/schemaFor`](this.value);
+
+    const hash = await allHash({
+      clusterReports:         this.value.getReports(),
+      // Ensure the clusterscan model has everything it needs
+      hasScheduledScanConfig: fetchSpecsScheduledScanConfig(schema),
```

Comment:
Not specific to this but has there been any talk or plans to slim the schemas further by having some global definitions that are expected to be loaded at the start?

It seems like half the definition size for clusterscan is for apimachiner definitions which I expect will be common to practically all schemas.

![image](https://github.com/rancher/dashboard/assets/55104481/9c68a181-8a54-4fed-8d0f-2f7cce834ef3)